resources/views/includes/script.blade.php

```diff
@@ -12,15 +12,28 @@ function bootstrapSupportBubble(element) {
             .addEventListener('click', () => {
                 const opening = container.classList.contains('opacity-0');
 
-                if (opening) {
-                    responseContainer.style.display = 'none';
-                    formContainer.style.display = 'block';
-                    container.classList.remove('translate-x-full', 'opacity-0');
-                    container.classList.add('translate-x-0', 'opacity-100');
-                } else {
-                    container.classList.remove('translate-x-0', 'opacity-100');
-                    container.classList.add('translate-x-full', 'opacity-0');
-                }
+                @if(config('support-bubble.direction') == 'rtl')
```

Comment:
I wonder if this can be rewritten without the need for so much duplication. Maybe put some class names in variables and play with that?

Also, always use strict comparisons.